_projects/2023/100481433.Rmd

```diff
@@ -0,0 +1,279 @@
+
+## Definitions
+
+**Types of Democracy in the Lexical Index:**
+
+1.  **Non-Electoral Autocracy:** A political system characterized by a single entity or a small group holding power without elections, often with a single autocrat or ruling elite.
+
+2.  **One-Party Autocracy:** A system where a single political party maintains absolute control over the government, suppressing the opposition and competitive elections.
+
+3.  **Multi-Party Autocracy without Elected Executive:** A political system with multiple parties, but the executive leader is not elected democratically.
+
+4.  **Multi-Party Autocracy:** Features multiple political parties, but the political system still retains autocratic elements.
+
+5.  **Exclusive Democracy:** Inclusive of elections but excludes certain groups or individuals from political participation.
+
+6.  **Male Democracy:** A democracy where only male citizens are allowed to participate in elections.
+
+7.  **Electoral Democracy:** A political system with regular elections where citizens can choose their leaders, but this may not guarantee full democratic rights.
+
+8.  **Polyarchy:** A comprehensive democracy with competitive elections, civil liberties, political pluralism, and broad participation.
+
+These definitions reflect the categorization of political systems within the Lexical Index, based on specific characteristics and levels of inclusiveness attained from Skaaning et al. (2015), with "Polyarchy" representing the highest level of democracy and inclusivity.
+
+## Getting the data
+
+The dataset, obtained from Our World in Data and based on research by Skaaning et al. (2015), categorizes the world's nations into democracies and autocracies. This graphic dynamically captures the shifts in these counts from 1789 through 2022, offering insights into the evolving landscape of global political systems nowadays. All the data needed for this chart can be downloaded directly from the Our World in Data website in the topic Democracy (<https://ourworldindata.org/grapher/countries-democracies-autocracies-lexical>).
+
+### **Library and the data set with the frequency variables**
+
+```{r}
+data <- read.csv("C:/Users/sofia/OneDrive/Documentos/Master Computational Social Sciences/Data Visualization/Graph/Data World csv.csv", 
+                 sep = ";", 
+                 dec = ",")
+library(ggplot2)
+library(extrafont)
```

Comment:
Is this library needed? Remove this if not, please.

Also, I don't see any code to load the fonts from Google Fonts. That should be added, right?